### ISSUE TITLE
Ideasbank- cannot post ideas

### ISSUE THREAD

**Issue by EcologyNorth** [IserveU/IserveU#795]
I cannot create a new idea. I've tried various points of access. When I attempt to save new content (new idea) and press the "save" button I get an error message. None of the content saves however a draft file was created (without any information uploaded). I have also tried updating the draft with no success.

**Comment by psaunders** [comment]
Does the account have the permissions (like is it approved etc)?

We're working on a new patch at the moment which is holding us up from rolling out an update if there is a bug.

**Comment by EcologyNorth** [comment]
The account being used is the main admin account, which is the same account
we have posted ideas from before. I just double checked and the profile
identifies it as a Full Administrator. I have also tried to post from my
personal account which also prompts the same error message after trying to
save -  "Sorry, something went wrong, RESTART".

Also note:
- the same error message appears as soon as I simply login as well
- I've been kicked back to the login prompt a few times while simply
navigating the webpages.


~Catherine

Administrative Team
admin@ecologynorth.ca

**Note that we've recently moved. Our new location is:*
Ecology North
5016 50 Avenue, Lower Level
PO Box 1684
Yellowknife, NT X1A 2P3

*Phone*: (867) 873-6019
*Fax*: (867) 873-6149
